### PR TITLE
Enhancement/months and years components

### PR DESCRIPTION
**Pull request by proand**

### CODE REVIEW

File: projects/fhi-angular-components/CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
 # Unreleased
 
 > Nov 7, 2023
-> Nov 7, 2023
-
-<!-- NB! Updates to FhiAutosuggest is not ready for release -->
-* :boom: **Breaking** Updates to `FhiAutosuggest`
 
+* :boom: **Breaking** Update `FhiAutosuggest` output from `selectedItemChange` to `itemSelectChange`, and update README
```
Comment (yleaxeman) [inline projects/fhi-angular-components/CHANGELOG.md:5]: `itemSelectChange` or `itemSelect`?

File: src/MOCK_DB_DATA/library-items/library-items-shared-data.ts [hunk 2]
```diff
@@ -181,9 +173,17 @@ export const LibraryItemsSharedData: LibraryItemsShared = {
...
+    id: 'time-selector-year-month',
+    title: 'Year month',
+  },
+  TimeSelectorYearMonthRange: {
+    id: 'time-selector-month-range',
+    title: 'Month range',
+  },
+  TimeSelectorYears: {
+    id: 'time-selector-years',
+    title: 'Years',
```
Comment (yleaxeman) [inline src/MOCK_DB_DATA/library-items/library-items-shared-data.ts:186]: Year, singular? Or will this be a multiselect eventually?

**Review by yleaxeman — APPROVED** [review]

File: projects/fhi-angular-components/CHANGELOG.md (hunk 1 above)
Comment (proand) [reply projects/fhi-angular-components/CHANGELOG.md:5]: Good question: it's both

File: src/MOCK_DB_DATA/library-items/library-items-shared-data.ts (hunk 2 above)
Comment (proand) [reply src/MOCK_DB_DATA/library-items/library-items-shared-data.ts:186]: Yes, will become a multiselect

**Review by yleaxeman — APPROVED** [review]

**Pull request merged**